pkg/kubelet/kubeletconfig/configsync.go

```diff
@@ -181,3 +205,43 @@ func latestNode(store cache.Store, nodeName string) (*apiv1.Node, error) {
 	}
 	return node, nil
 }
+
+// eventf constructs and returns an event containing a formatted message
+// similar to k8s.io/client-go/tools/record/event.go
+func eventf(nodeName, eventtype, reason, messageFmt string, args ...interface{}) *apiv1.Event {
```

Comment:
nit: any reason to use `eventtype` over `eventType` ?

Reply:
No good ones. I think I probably saw it cased that way somewhere else. I'll change it.

Reply:
Updated